### ISSUE TITLE
vue-validator doesn't accept partials from vm object

### ISSUE THREAD

**Issue by sqal** [kazupon/vue-validator#209]
### vue & vue-validator version

1.0.21, 2.0.2
### Steps to reproduce

```
<div class="form-errors">
  <validator-errors :validation="$validation" partial="error"></validator-errors>
</div>

// inside our instance

{
  partials: {
    error: '{{ message }}'
  }
}
```
### What is Expected?

Vue-validator to accept partial template from **partials** property inside instance object.
### What is actually happening?

Console throws an error:

`[Vue warn]: Failed to resolve partial: error (found in component: <validator-error>)`

Seems like partial must be declared using Vue.partial method only..

**Comment by kazupon** [comment]
This is the spec.
I'll try to improve.

**Comment by kazupon** [comment]
I'll try to implement at v3.0.

**Comment by kazupon** [comment]
released v3.0.0-alpha.1
https://github.com/vuejs/vue-validator/releases/tag/v3.0.0-alpha.1

In 3.0 later, validator-errors component is deprecated.
We can implement with using `errors` validation properties and component.
